### PR TITLE
Add regression test for #19739

### LINKED ISSUE #19739: Limit with negative base
`(-1/4)**x` as `x->oo`
Wolframalpha gives `0`, sympy raises `NotImplementedError: Result depends on the sign of sign(-log(4) + I*pi)`.
Why is that and what is the correct behavior?

### PR DESCRIPTION
**Pull request by aaryandewan**
#### Release Notes

NO ENTRY
Fixes #19739

### CODE REVIEW

File: sympy/series/tests/test_limits.py [hunk 1]
```diff
@@ -845,3 +845,6 @@ def test_issue_13715():
 
 def test_issue_15055():
     assert limit(n**3*((-n - 1)*sin(1/n) + (n + 2)*sin(1/(n + 1)))/(-n + 1), n, oo) == 1
+
```
Comment (sachin-4099) [inline sympy/series/tests/test_limits.py:848]: Please maintain two line spacing.

File: sympy/series/tests/test_limits.py [hunk 2]
```diff
@@ -845,3 +845,6 @@ def test_issue_13715():
...
+def test_issue_19739():
+    assert limit((-1/4)**n, n, oo) == 0
```
Comment (sachin-4099) [inline sympy/series/tests/test_limits.py:850]: This should be `assert limit((-S(1)/4)**x, x, oo) == 0`. `x` should be used and not `n`.

**Comment by sachin-4099** [comment]
The Release notes should have `NO ENTRY`.

**Comment by sachin-4099** [comment]
Looks good to me. This can be merged @jksuom.

**Comment by Smit-create** [comment]
@aaryandewan  Add `Fixes #issue_number` in the release notes so that the issue gets closed upon merging this automatically

**Pull request merged**